docs/40-troubleshooting.md

```diff
@@ -1,4 +1,64 @@
-# Getting logs without Azure Portal
+## Debugging and Troubleshooting
+
+### After creating my cluster, my nodes go to a 'startTaskFailed' state. Why?
+The most common case for this is that there was an issue with package installation or the custom script failed to run. To troubleshoot this you can simply download the output logs from the node.
+
+The following 2 nodes failed while running the start task:
+tvm-769611554_1-20170912t183413z-p
+tvm-769611554_2-20170912t183413z-p
+
+
+```r
+cluster <- makeCluster('myConfig.json')
```

Comment:
can we move this line to line 24? .. we don't use the cluster variable until we actually runt he 'getClusterFile' function

Reply:
Yes BUT this method should print out any known errors. If not, you can use the listPoolNodes() method. Is the way I have it laid out too confusing? I can rethink it, but makeCluster should be the first thing... perhaps it should be a separate troubleshooting item?